docker/local-pytest/up-backend.sh

```diff
@@ -18,5 +18,6 @@ docker run -it \
   -e "GUNICORN_TIMEOUT=86400" \
   -e "APP_MODULE=pdp.wsgi:backend" \
   -v $(pwd):/codebase \
+  -v /storage/data:/storage/data:ro \
   --name pdp-local-pytest_backend \
-  pdp-local-pytest
\ No newline at end of file
+  pcic/pdp-local-pytest
```

Comment:
This and `down-backend` seem like useful commands, but not documented anywhere. Could you please mention them in documentation somewhere? Their use is fairly obvious, but the PDP test setup is complicated enough that I am in favour of documenting everything.